OrbitCore/SamplingProfiler.cpp

```diff
@@ -319,6 +319,26 @@ void SamplingProfiler::ProcessSamples() {
   m_State = DoneProcessing;
 }
 
+void SamplingProfiler::UpdateSampledFunctions() {
+  for (auto& data_entry : m_ThreadSampleData) {
+    ThreadSampleData& thread_sample_data = data_entry.second;
+    std::vector<SampledFunction>& sampled_functions =
+        thread_sample_data.m_SampleReport;
+    for (SampledFunction& sampled_function : sampled_functions) {
+      Function* function =
+          m_Process->GetFunctionFromAddress(sampled_function.m_Address, false);
+      if (function == nullptr) {
+        continue;
+      }
+
+      const std::string& function_name = function->PrettyName();
```

Comment:
The logic to fill `Capture::GAddressToFunctionName` initially is currently in `SamplingProfiler::AddAddress`. Can that be reused somehow? First, because it's actually different, giving the precedence to `LinuxAddressInfo`s (not saying it's better, that can be changed), and second to have it in a single place.

Reply:
hmm. Maybe we should give a function->PrettyName precedence. are functions in adress_info unmangled?

Reply:
Yes, the are unmangled, all the ones that you see without loading any module come from there. Still, maybe `function->PrettyName` should have the precedence. My point was about being consistency, ideally by reusing the same code.

Reply:
done.